_docs/archived_modules.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: Archived Modules
+date: 2019-11-29
+summary: Vox Pupuli policy on archiving modules
+---
+
+It is Vox Pupuli's mission to provide a home for modules so that development can continue after the original authors/maintainers have moved on.
+In rare circumstances a module may be archived and placed into a read-only state.  This typically will only occur if the module/project no longer serves a valid
+purpose.  For example, a module that interacted with a discontinued 3rd party service.
+
+If you come across an archived Vox Pupuli project that you are still using or think should not have been archived, [let us know][email].
+The decision to archive a module isn't necessarily final and can be undone.
+
+[email]: https://help.github.com/en/articles/generating-a-new-gpg-key
```

Comment:
Looks like an incorrect URL for this

Reply:
Oops

Reply:
Fixed (and renamed to accommodate policy on deprecations etc)